virttest/utils_cpi.py

```
@@ -0,0 +1,934 @@
+                "CPI_SYSTEM_TYPE",
+                "CPI_SYSTEM_NAME",
+                "CPI_SYSPLEX_NAME",
+                "CPI_PERMIT_ON_PVGUEST",
+            ]:
+                if key in config_dict:
+                    value = config_dict[key]
+                    new_config_lines.append(f'{key}="{value}"')
+
+            new_config_content = "\n".join(new_config_lines)
+
+            temp_file = "/tmp/cpi_config_new"
```

Ruff (0.13.3)

Probable insecure usage of temporary file or directory: "/tmp/cpi_config_new"

(S108)